fusionauth/resource_fusionauth_tenant.go

```diff
@@ -1220,6 +1220,12 @@ func newExternalIdentifierConfiguration() *schema.Resource {
 				ValidateFunc: validation.IntAtLeast(1),
 				Description:  "The time in seconds until a two factor Id is no longer valid and cannot be used by the Two Factor Login API. Value must be greater than 0.",
 			},
+			"two_factor_one_time_code_id_time_to_live_in_seconds": {
+				Type:         schema.TypeInt,
+				Required:     true,
```

Comment:
Should this be `Optional: true` instead of `Required: true`? This is documented as optional in the FusionAuth API, and optional in the above `.md` file. 

- https://fusionauth.io/docs/v1/tech/apis/tenants#create-a-tenant